# Applies C++ name mangling to all of PortableGL's functions so that they don't conflict with the functions in Emscripten's OpenGL implementation.
# Otherwise, the Emscripten build crashes on startup because the Emscripten version of RetroArch also depends on OpenGL, and it will incorrectly try to use our OpenGL implementation instead of Emscripten's.

--- a/portablegl.h
+++ b/portablegl.h
@@ -296,9 +296,9 @@ IN THE SOFTWARE.
 #define GL_H
 
 
-#ifdef __cplusplus
-extern "C" {
-#endif
+
+
+
 
 
 
@@ -3299,9 +3299,9 @@ void put_wide_line_simple(Color the_color, float width, float x1, float y1, floa
 void put_triangle(Color c1, Color c2, Color c3, vec2 p1, vec2 p2, vec2 p3);
 
 
-#ifdef __cplusplus
-}
-#endif
+
+
+
 
 // end GL_H
 #endif
